src/main/java/net/education/kyivstar/config/CreateSchema.java

```diff
@@ -0,0 +1,71 @@
+package net.education.kyivstar.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.DATABASE_PATH;
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.TABLES_PATH;
+import static net.education.kyivstar.services.util.Utils.readSqlScriptFromFileStream;
+
+
+public class CreateSchema {
+    private EmbeddedMariaDbLifeCycle embeddedMariaDbLifeCycle;
```

Comment:
Field 'embeddedMariaDbLifeCycle' may be 'final'

Reply:
+